### PR TITLE
[search] Fixed features sorting for viewport search.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

File: search/search_query.cpp [hunk 1]
```diff
@@ -125,16 +125,25 @@ class IndexedValue : public search::IndexedValueBase<Query::kQueuesCount>
...
 
 public:
   explicit IndexedValue(unique_ptr<impl::PreResult2> v)
-    : m_val(move(v)), m_rank(m_val ? m_val->GetRankingInfo().GetLinearModelRank() : 0)
+    : m_val(move(v)), m_rank(0), m_distanceToPivot(numeric_limits<double>::max())
   {
+    if (!m_val)
+      return;
+
+    auto info = m_val->GetRankingInfo();
```
Comment (ukolshurika) [inline search/search_query.cpp]: DO NOT MERGE THIS11111

Comment (mpimenov) [inline search/search_query.cpp]: ok

Comment (ygorshenin) [inline search/search_query.cpp]: This should be replaced by GetLinearModeRank. No hurry, I'm going to fix this tomorrow.

Comment (ygorshenin) [inline search/search_query.cpp]: Done

**Comment by ygorshenin** [comment]
@mpimenov PTAL

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**